.github/workflows/tests.yaml

```diff
@@ -248,56 +248,56 @@ jobs:
-      # is ok as long as the test doesn't
-      # include newer incompatible features
-      - run: sh -c "$(curl -sSfL https://release.solana.com/v1.8.14/install)"
-        shell: bash
-      - run: echo "/home/runner/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
-        shell: bash
-      - run: solana-keygen new --no-bip39-passphrase
-        shell: bash
-      - run: solana config set --url localhost
-        shell: bash
-      - uses: actions/download-artifact@v2
-        with:
-          name: anchor-binary
-          path: ~/.cargo/bin/
-      - run: chmod +x ~/.cargo/bin/anchor
+  # # this test exists to make sure that anchor
+  # # checks rent correctly for legacy accounts
+  # # that don't have to be rent-exempt
+  # test-misc-non-rent-exempt:
+  #   # the anchor cli is built with a different solana version
+  #   # but that's fine since it's just the cli
+  #   needs: setup-anchor-cli
+  #   name: Test tests/misc/nonRentExempt
+  #   runs-on: ubuntu-18.04
+  #   timeout-minutes: 30
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: ./.github/actions/setup/
+  #     - uses: ./.github/actions/setup-ts/
+  #     - uses: actions/cache@v2
+  #       name: Cache Solana Tool Suite
+  #       id: cache-solana
+  #       with:
+  #         path: |
+  #           ~/.cache/solana/
+  #           ~/.local/share/solana/
+  #         key: solana-${{ runner.os }}-v0000-1.8.14
+  #     # using an outdated validator but that
+  #     # is ok as long as the test doesn't
+  #     # include newer incompatible features
+  #     - run: sh -c "$(curl -sSfL https://release.solana.com/v1.8.14/install)"
+  #       shell: bash
+  #     - run: echo "/home/runner/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
+  #       shell: bash
+  #     - run: solana-keygen new --no-bip39-passphrase
+  #       shell: bash
+  #     - run: solana config set --url localhost
+  #       shell: bash
+  #     - uses: actions/download-artifact@v2
+  #       with:
+  #         name: anchor-binary
+  #         path: ~/.cargo/bin/
+  #     - run: chmod +x ~/.cargo/bin/anchor
 
-      - uses: actions/cache@v2
-        name: Cache tests/misc target
-        id: cache-test-target
-        with:
-          path: tests/misc/target
-          key: cargo-${{ runner.os }}-tests/misc-${{ env.ANCHOR_VERSION }}-1.8.14-${{ hashFiles('**/Cargo.lock') }}
+  #     - uses: actions/cache@v2
+  #       name: Cache tests/misc target
+  #       id: cache-test-target
+  #       with:
+  #         path: tests/misc/target
+  #         key: cargo-${{ runner.os }}-tests/misc-${{ env.ANCHOR_VERSION }}-1.8.14-${{ hashFiles('**/Cargo.lock') }}
 
-      - run: cd tests/misc && yarn --frozen-lockfile
-      - run: cd tests/misc
-      - run: cd tests/misc && chmod +x ci.sh && ./ci.sh
-      - run: cd tests/misc && anchor test --skip-lint
+  #     - run: cd tests/misc && yarn --frozen-lockfile
+  #     - run: cd tests/misc
```

Comment:
Can we disable the individual test that's failing? These misc tests are important to run.

Reply:
the programs get built fine and `./tests/misc/misc.ts` [passes](https://github.com/coral-xyz/anchor/runs/7964708081?check_suite_focus=true). its only for the [this test step](https://github.com/coral-xyz/anchor/runs/7964706609?check_suite_focus=true) for the `nonRentExempt` that uses an older version of Solana that fails to build and never gets to run the test.

Reply:
Got it. Thanks for the clarification.